package.json

```diff
@@ -29,11 +29,70 @@
       "no-shadow": [
         "error",
         {
-          "allow": ["resolve"]
+          "allow": [
+            "resolve"
+          ]
         }
       ],
       "no-use-before-define": "warn",
       "import/no-unresolved": "warn"
     }
+  },
+  "dependencies": {
+    "underscore": "^1.9.2"
```

Comment:
This is not how you update dependencies in Meteor.js. Adding here is not allowed and has no impact on what you are claiming to solve.